### PR TITLE
Add content-highlight plugin.

### PR DESCRIPTION
**Pull request by moniika**
### Resolves
https://github.com/google/blockly-samples/issues/312

### Summary
Adds content-highlight plugin to plugins.

### Screenshots
58a261fd-0600-4793-b4fc-fc974a991495
#### Changing themes
image
image

### Additional Info
The current implementation has issue with potential out of memory crashes if there is a lot of scrolling or dragging occurring due to a backup of ViewportChange events on `Blockly.Events.FIRE_QUEUE_`. The change https://github.com/google/blockly/pull/4792 introduces squashing consecutive ViewportChange events which should fix this issue.

### CODE REVIEW

File: plugins/content-highlight/package.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "workspace"
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/master/content-highlight#readme",
+  "bugs": {
+    "url": "https://github.com/google/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/blockly-samples.git",
+    "directory": "content-highlight"
```
Comment (rachel-fenichel) [inline plugins/content-highlight/package.json]: plugins/content-highlight

File: plugins/content-highlight/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# workspace-content-highlight [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
```
Comment (rachel-fenichel) [inline plugins/content-highlight/README.md:1]: (Can be a followup) Please add an image to show what this one does--it's much clearer than a written description.

**Review by rachel-fenichel — APPROVED** [review]
Looks good!

File: plugins/content-highlight/README.md (hunk 2 above)
Comment (moniika) [reply plugins/content-highlight/README.md:1]: Done.

**Pull request merged**